packages/discount-code-exporter/src/main.js

```diff
@@ -137,17 +137,23 @@ export default class DiscountCodeExport {
     }).discountCodes
   }
 
-  // Use this method to make the `cartDiscounts`
+  // Use this method to make the `cartDiscounts` and `groups`
   // field compatible with the importer
   _processCode(data: CodeData): Object {
-    const newCodeObj = data
-    const cartDiscounts = newCodeObj.cartDiscounts.reduce(
+    const { cartDiscounts, groups, ...restCodeData } = data
+    const cartDiscountsString = cartDiscounts.reduce(
       (acc: string, discount: Object): string => {
         if (!acc) return discount.id
         return `${acc}${this.config.multiValueDelimiter}${discount.id}`
       },
       ''
     )
+    const groupsString = groups
+      ? groups.reduce((acc: string, group: string): string => {
```

Comment:
Whenever I read a `reduce` with `acc` I wonder what `acc` actually is. In our case `joinedGroups`?

Reply:
I just have the `acc` name stuck in my head no matter what 😄